### PR TITLE
fix(core/utils): allow lang aliases with getIntlData

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: src/core/utils.js [hunk 1]
```diff
@@ -292,12 +292,24 @@ export function norm(str) {
   return str.trim().replace(/\s+/g, " ");
 }
 
+/**
+ * @param {string} lang
+ */
+function resolveLanguageAlias(lang) {
+  const aliases = {
+    "zh-hans": "zh",
+    "zh-cn": "zh",
```
Comment (saschanaz) [inline src/core/utils.js]: Has there been a request for this?

Comment (sidvishnoi) [reply src/core/utils.js]: No. But it existed in core/l10n.js, so decided to keep. I think we can ignore this. (Can always add later if someone needs it)

Comment (sidvishnoi) [reply src/core/utils.js]: A search through github.com/W3C found only one document using it: https://github.com/w3c/danmaku

Comment (saschanaz) [reply src/core/utils.js]: Ah, I agree to keep it then. Interesting API BTW.

Comment (sidvishnoi) [reply src/core/utils.js]: Marcos would've asked for a switch-case :stuck_out_tongue_closed_eyes:

Comment (saschanaz) [reply src/core/utils.js]: I would've asked for generalized algorithm for `ko-kr` etc, but it's just an existing code, no need to make it complex.

**Comment by sidvishnoi** [comment]
@saschanaz Release v25.0.3 after this?

**Comment by saschanaz** [comment]
Sounds good 👍

But you do 😂

**Pull request merged**